docs/julia.rst

```diff
@@ -328,15 +336,17 @@ The following functions must be available:
 +---------------------------------------------------------------------------------------------------------------+
 | mulmodu256(x:u256, y:u256, m:u256) -> z:u256| (x * y) % m with arbitrary precision arithmetics                |
 +---------------------------------------------------------------------------------------------------------------+
-| ltu256(x:u256, y:u256) -> z:bool            | 1 if x < y, 0 otherwise                                         |
```

Comment:
Should we be going for `ltu256` or `lt_u256`?

Reply:
I think if we change it for `lt`, we have to change it for everything. I'm not really sure what the best option is, though ...

Reply:
I like `lt.u256` for the looks, but it goes against the grammar we have.

Reply:
Also how do you feel about allowing `#` in identifiers in julia?

Reply:
I would actually prefer `.`.

Reply:
Doesn't llvm also use `.`?

Reply:
I thought we agreed on `.` already. `#` was in addition.

Reply:
ah sorry.